pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Validate AsyncAPI Documents"
+description: In this guide, we'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+# Introduction
+In this guide, we'll learn multiple ways to validate AsyncAPI documents. 
+
+When people talk about AsyncAPI validation, sometimes they mean completely different things: **validation of AsyncAPI documents** or **validation of messages in runtime** _(message schemas provided in the AsyncAPI document)_.  
+
+# Prequisites
+Because this guide covers multiple options for validating AsyncAPI documents, there are different tool options you might choose to install:
+- [AsyncAPI CLI](https://github.com/asyncapi/cli#installation)
+- [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation)
+
+# Create AsyncAPI documents
+An AsyncAPI document is a machine-readable definition of your Event-Driven API that can be used to generate documentation and code. You can create an AsyncAPI document with the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document requires two different pieces:
```

Comment:
it is not much about requirement

it is more about what people look for, so this section (or maybe intro should be extended) should rather say: "when people talk about validating AsyncAPI documents, they can mean 2 things, making sure it is correctly following AsyncAPI specification or it is correctly following the best practices or company governance rules"

so yeah, user is not required to do best practice validation (many do not care, they do not have them).

also maybe for SEO also push here somehow another phrase people use for best practice/governance validation, which is `lint`. People also say, `is there a linter for AsyncAPI`, so they have best practices linting included.